frontend/public/components/dashboard/dashboards-page/overview-dashboard/activity-card.tsx

```diff
@@ -24,20 +24,16 @@ import {
 import { uniqueResource } from './utils';
 import { PrometheusResponse } from '../../../graphs';
 import { connectToFlags, WithFlagsProps, FlagsObject } from '../../../../reducers/features';
+import { useK8sWatch } from '../../../utils/k8s-watch-hook';
 
-const eventsResource: FirehoseResource = { isList: true, kind: EventModel.kind, prop: 'events' };
+const eventsResource: FirehoseResource[] = [
+  { isList: true, kind: EventModel.kind, prop: 'events' },
+];
 
-const RecentEvent = withDashboardResources(
```

Comment:
So basically the new `useK8sWatch` hook can also replace `withDashboardResources` HOC.

This really simplifies the code.

Maybe we can get rid of the `withDashboardResources` HOC entirely using this new hook?

Reply:
It replaces just part of the `withDashboardResources`. Once the hook is merged, we can remove that and every card component can use the hook